.eslintrc.js

```diff
@@ -24,7 +27,19 @@ module.exports = {
     'vue/html-self-closing': 'error',
     'vue/mustache-interpolation-spacing': 'error',
   },
+
   parserOptions: {
     parser: 'babel-eslint',
   },
+
+  overrides: [
```

Comment:
I'm not sure we need this section, since we already have ESLint files in our test folders (`tests/unit/.eslintrc.js`), and this references a non-existent `__tests__` folder.

Reply:
I'm not sure if it's needed or not. To be safe, I updated it to match the unit tests directory and we'll see what happens.